### ISSUE TITLE
Sort statistics or other columns ascending or descending

### ISSUE THREAD

**Issue by mwpowellhtx** [dedmen/ArmaScriptProfiler#20]
Such as:
![image](https://user-images.githubusercontent.com/1824187/113514175-1f8ee680-953b-11eb-9372-5cca8cafb408.png)

**Comment by dedmen** [comment]
I don't understand this ticket?
You want them to sort? Just click on it and it sorts.
I don't understand.

**Comment by mwpowellhtx** [comment]
Sort *_ASCENDING_* as well as descending.

**Comment by dedmen** [comment]
Yes. I still don't understand what you want?
if you want to change sort direction, just click again.

**Comment by mwpowellhtx** [comment]
I did, does not sort in the other direction.

**Comment by mwpowellhtx** [comment]
Possibly similar in nature to when profiling "stops", i.e. Arma instance vanishes. Should be able to click "stopped" then presumably "resume" but that does not work either.

**Comment by Fusselwurm** [comment]
uhm isnt this an issue with Tracy instead of ArmaScriptProfiler

**Comment by mwpowellhtx** [comment]
> uhm isnt this an issue with Tracy instead of ArmaScriptProfiler

Yes, quite probably, but I was not aware of a `Tracy` specific repo at the time.
